### ISSUE TITLE
How to train on A Single GPU

### ISSUE THREAD

**Issue by BoomShakaY** [guxinqian/Simple-CCReID#8]
Thanks for your contribution, but by reproducing your code I found that it only works for two or more GPUs. I wonder if you could guide me on what changes I should make if I want to train on a single GPU, thanks a lot again!

**Comment by guxinqian** [comment]
You can double config.DATA.TRAIN_BATCH in configs and change --nproc_per_node and --gpu in script.sh to 1.
Note that the results on single GPU may be lower that the results on two GPUs, since the optimal batch size for BatchNorm on small datasets is 32 per GPU.

**Comment by BoomShakaY** [comment]
Thanks for your quick reply!
